src/Apps/Settings/Routes/Orders/Components/SettingsOrdersRow.tsx

```diff
@@ -55,7 +55,7 @@ interface OrderActionButtonProps {
 
 const ORDER_ICONS: Record<string, React.ReactElement> = {
   APPROVED: <PendingIcon fill="mono100" />,
-  CANCELLED: <CloseStrokeIcon fill="red100" />,
+  CANCELED: <CloseStrokeIcon fill="red100" />,
```

Comment:
You could probably make the keys of this record into the actual prop type: `SettingsOrdersRow_order$data['buyerState']` for a little better type help